strategies.py

```diff
@@ -74,7 +78,7 @@ def __init__(self, network, seconds_per_move=5, simulations_per_move=0,
         if two_player_mode:
             self.temp_threshold = -1
         else:
-            self.temp_threshold = TEMPERATURE_CUTOFF
+            self.temp_threshold = FLAGS.softpick_move_cutoff
```

Comment:
unrelated temp_threshold appears to be used inconsistently

https://github.com/tensorflow/minigo/blob/master/strategies.py#L135
https://github.com/tensorflow/minigo/blob/master/strategies.py#L156

I think that L156 should be >=

Reply:
it was fixed in the v5-running head.

Reply:
done